bin/desi_proc

```diff
@@ -154,14 +154,14 @@ else:
     #- Fill in values from raw data header if not overridden by command line
     fx = fitsio.FITS(args.input)
     if 'SPEC' in fx:                                #- 20200225 onwards
-        hdr = fits.getheader(args.input, 'SPEC')
-        #hdr = fx['SPEC'].read_header()
+        #hdr = fits.getheader(args.input, 'SPEC')
+        hdr = fx['SPEC'].read_header()
     elif 'SPS' in fx:                               #- 20200224 and before
-        hdr = fits.getheader(args.input, 'SPS')
-        #hdr = fx['SPS'].read_header()
+        #hdr = fits.getheader(args.input, 'SPS')
+        hdr = fx['SPS'].read_header()
     else:
-        #hdr = fits.getheader(args.input, 0)
-        hdr = fx[0].read_header()
+        hdr = fits.getheader(args.input, 0)
+        #hdr = fx[0].read_header()
```

Comment:
I agree with the changes in the "if" and "elif" blocks to use fitsio instead of astropy, but this final "else" should be left as is, i.e. using `fx[0].read_header()` (fitsio) instead of `fits.getheader` (astropy).

Context for others seeing this ticket: apparently astropy 4 FitsHeader objects can't be pickled by mpi4py for a broadcast, which broke desi_proc in the 20.7 software release.  Now that fitsio 1.x uses strings instead of bytes, it is easier to use fitsio for reading the headers, broadcasting, and using downstream.

Reply:
Agree :-) Leave the final 'else' as is.